openqa-trigger-bisect-jobs

```diff
@@ -311,6 +311,7 @@ def main(args):
             )
         except subprocess.SubprocessError as err:
             if 'the repositories for the below updates are unavailable' in str(err.stderr):
+                openqa_comment(job_id, base_url, err.stderr, args.dry_run)
```

Comment:
I think users would appreciate a bit more explanation here, e.g. "Not triggering any bisect jobs because: {err.stderr}"

Reply:
Yes, just stderr without context is not helpful.

Reply:
Also this would add another comment but what we should do is only save some extra text that goes into the final comment that is written anyway in line [L330](https://github.com/os-autoinst/scripts/pull/378/files#diff-cd273835e66c0f36337b4df0b3a844b8d5486cf47b5ec0733c4479906c6871a9L330)

Reply:
I see. In that case disregard my former comment.